src/components/src/index.ts

```diff
@@ -241,6 +241,9 @@ export {
   LayerTitleSectionFactory
 } from './side-panel/layer-panel/layer-panel-header';
 
+import {LayerTypeListItemType} from './side-panel/layer-panel/layer-type-list-item';
+export type {LayerTypeListItemType};
```

Comment:
`export type {LayerTypeListItemType} from '../side-panel/layer-panel/layer-type-list-item';`

Reply:
yeah, tried this one as well.
for some reason moving this line down fixes the breaking issue from prettier.